src/lib/Validator/Constraints/UniqueContentTypeIdentifier.php

```diff
@@ -20,6 +22,14 @@ class UniqueContentTypeIdentifier extends Constraint
      */
     public $message = 'ez.content_type.identifier.unique';
 
+    public static function getTranslationMessages()
```

Comment:
This needs `: array` return type and PHPDoc array type specification (`@return array<\PROPER_FQCN\Message>`).